packages/origin-ui-core/src/styles/variables.scss

```diff
@@ -21,6 +21,8 @@ $createBundleCertificatesGroupBg: #3b3b3b;
 $createBundleCertificatesGroupHeaderBg: #373737;
 $createBundleCertificateForSale: #303030;
 $createBundleSelectedCertificate: #3a2146;
+$filterBg: #3a3a3a;
+$filterItemBg: #2d2d2d;
```

Comment:
**Do we really need these many new colors**? Can't we replace most of these colors with a few colors which can be reused?

1. Are `$filterBg` and `$createBundleCertificatesGroupBg` really that different that they have to be defined differently?
2. `$bundleDetailsBg` and `$bundleCardBorderBg` are basically the **same color**, why are they defined twice?
3. `$filterItemBg` and `$createBundleCertificatesBg` are the **same color**.

Can't these just be replaced with one color called `$secondaryColor: #3b3b3b`?

Then we can create different colors such as `$secondaryColorDarker: darken($secondaryColor, 10%)`.

Adding a new color for each new component that we build is stupid. We should have a set number of colors in the whole application, which we then reuse, and that can be easily customized.

I suggest a similar approach to [Boostrap](https://getbootstrap.com/docs/4.0/utilities/colors/#background-color), where we can define a **primary** (e.g. purple) and **secondary** color (e.g. grey), and then all other variations of that color is dynamically generated using `darken()` and `lighten()` variations of the primary and secondary colors.

Does this make sense?

Reply:
Agree. I'll try to implement this approach